### ISSUE TITLE
Listing name ('pkg_joommyname') starts with extension type prefix

### ISSUE THREAD

**Issue by anibalsanchez** [joomla-extensions/jedchecker#153]
Hi @dryabov,

I don't understand what is being reported here. The install looks fine as the rest of the extensions in the package.

```
Listing name ('pkg_joommyname') starts with extension type prefix
```

```xml
<?xml version="1.0" encoding="UTF-8" ?>
<extension type="package" version="3.0" method="upgrade" client="site">
	<name>pkg_joommyname</name>
....
```

Is the translation label missing?

**Comment by Llewellynvdm** [comment]
@anibalsanchez please also state what file what line? it makes it easier for all to be on the same page.

**Comment by anibalsanchez** [comment]
![slot-error](https://user-images.githubusercontent.com/385780/132089163-46ccd2e0-f1be-4341-bcad-15a11b133a3c.png)

**Comment by dryabov** [comment]
pkg_joomslot1.0.zip package contains no language file, that's why the title is not translated and used as is.

**Comment by anibalsanchez** [comment]
Ok, cool.
